test/testPO.js

```diff
@@ -0,0 +1,16 @@
+import EnterLogPage from "../pages/startPage";
+import LoginPage from "../pages/loginPage";
+
+const USER = {
+  USERNAME: "nick98_molo@icloud.com",
+  PASSWORD: "Kol90407",
+};
+
+
+describe("Open Page", function () {
+  it("Open browser page in url and input login & password", async function () {
+    await browser.url("https://www.onliner.by/");
```

Comment:
Store URL as constant too
I would suggest to create `config.js` and store URL and USER in this file

Reply:
ANd rename test file to `login.e2e.test.js`